examples/data/e2e_tests/test_jsons/castingConformanceRuleTest.json

```diff
@@ -2,9 +2,9 @@
 {
     "vars": {
       "spark-submit": "spark-submit --num-executors 2 --executor-memory 2G --deploy-mode client",
-      "spark-conf": "--conf 'spark.driver.extraJavaOptions=-Denceladus.rest.uri=http://localhost:8080/rest_api/api -Dspline.mongodb.name=spline -Dspline.mongodb.url=mongodb://127.0.0.1:27017/ -Denceladus.recordId.generation.strategy=stableHashId'",
-      "enceladus-job-jar": "spark-jobs/target/spark-jobs-2.19.0-SNAPSHOT.jar",
-      "credentials": "--rest-api-credentials-file ~/.ssh/menasCredential.properties",
+      "spark-conf": "--conf 'spark.driver.extraJavaOptions=-Denceladus.rest.uri=http://localhost:8080/rest_api_war/api -Dspline.mongodb.name=spline -Dspline.mongodb.url=mongodb://127.0.0.1:27017/ -Denceladus.recordId.generation.strategy=stableHashId'",
```

Comment:
I think `rest-api` is also in the default resource files. Shouldn't it be consistent? 🤔

Reply:
Sure. "_war" removed from all files in commit 314a6b24073cae4a60931ed9885c319157142c84.